src/app/components/CourseChoice/CourseChoice.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./CourseChoice.module.css";
+
+type CourseChoiceProps = {
+  prioIcon: React.SVGProps<SVGSVGElement>;
+  courseChoice: string;
+  binIcon: React.SVGProps<SVGSVGElement>;
```

Comment:
The name tells me that it has to be the `binIcon` all the time. Is it possible that you have different bin icons?
If not, you can import the icon in this component.

Reply:
Done. Thank you!